File: 1672_Richest_Customer_Wealth/Solution.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        
        int max_res = Integer.MIN_VALUE;

        for (int i = 0; i < accounts.length; i++){
            int sum = 0;
            for (int j = 0; j < accounts[i].length; j++){
                sum += accounts[i][j];
            }
            max_res = Math.max(sum,max_res);
        }
        return max_res;
    }
}

File: 1672_Richest_Customer_Wealth/Solution.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        max_wealth = 0
        for i in range(0, len(accounts)):
            sum = 0
            for j in range(0,len(accounts[i])):
                sum += accounts[i][j]
            max_wealth = max(max_wealth, sum)
        return max_wealth 
        
            
